packages/apple-pay-integration/e2e/ApplePaySessionCustomerStepMockObject.ts

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+/* eslint-disable @typescript-eslint/no-empty-function */
+const addApplePaySessionToChrome = () => {
+    class ApplePaySessionCustomerStep {
+        [x: string]: any;
```

Comment:
Is this necessary? Also, we should avoid using `any` if possible.

Reply:
I've added this item to remove typescript errors on the Apple Pay methods that aren't declared on the file. eg.                 `this.onvalidatemerchant(validationEvent);`. I can just do a ts-ignore instead.

Reply:
I'll remove it as it's not part of the `ApplePaySession` definition

Reply:
Removed the key value type